css/mobile-first.css

```diff
@@ -10,13 +10,30 @@ https://coolors.co/96bbbb-faf3dd-c8d5b9-f2e3bc-696d7d
 
 /* Global styles
 ------------------------------*/
-html, body {
-    overflow-x: hidden; /* removed unwanted white space on right side of mobile view */
-}
 * {
   box-sizing: border-box;
 }
 
+h1 {
+    font-size: 60px;
+    line-height: 1;
+    font-weight: bold;
+  }
+
+h2 {
+    font-size: 30px;
+  }
+
+h1, h2 {
+  font-family: 'Baloo 2', cursive;
+  font-weight: 400;
+  margin: 0;
+}
```

Comment:
Typically, declarations that are applicable to multiple selectors go before the individual ones, so these should be moved to before the ones for `h1` and `h2`